Philips.CodeAnalysis.MaintainabilityAnalyzers/Philips.CodeAnalysis.MaintainabilityAnalyzers.md

```diff
@@ -74,3 +74,4 @@
 | PH2121  | Throw informational exceptions               | Specify context to a thrown exception, by using a constructor overload that sets the Message property. |
 | PH2122  | Avoid Exceptions from unexpected locations   | Avoid throwing exceptions from unexpected locations, like Finalizers, Dispose, Static Constructors, some operators, and overidden methods of Object. |
 | PH2123  | Pass sender to EventHandler                  | Prevent passing null values for sender/object to event handler (for instance-based events).|
+| PH2125  | Align number of + and == operators           | Align the number of operators + and ==, as these are often used in combination with each other. |
```

Comment:
The description is more a description of the implementation than what the rule is trying to accomplish. For example, consider changing to be closer to what TICS is saying. "Overload the equality operator (==), when you overload the addition (+) operator"

Reply:
I agree, done for all related